projects/cashmere/src/lib/app-switcher/app-switcher.service.spec.ts

```diff
@@ -7,20 +7,21 @@ describe('AppSwitcherService', () => {
     describe('when initialized with a discoveryServiceUri ending in a version', () => {
         let service: AppSwitcherService;
 
-        beforeEach(async(() => {
+        beforeEach(waitForAsync(() => {
             TestBed.configureTestingModule({
                 imports: [HttpClientTestingModule],
                 providers: [AppSwitcherService, {provide: APP_SWITCHER_CONFIG, useValue: {discoveryServiceUri: 'foo://bar/baz/v1'}}]
             });
         }));
 
-        beforeEach(() => (service = TestBed.get(AppSwitcherService)));
+        beforeEach(() => (service = TestBed.inject(AppSwitcherService)));
 
         it('should have the correct allApplicationsUri', () => {
             expect(service.allApplicationsUri).toBe('foo://bar/baz/apps');
         });
 
-        it('should use the correct uri when fetching services', async(
+        // TEST HAS NO EXPECTATIONS
+        xit('should use the correct uri when fetching services', waitForAsync(
```

Comment:
should we delete this guy?

Reply:
I figured these were stubbed in to be added at a later date - but who knows?

Reply:
I think the reason it reports there is no expectation is that the mock.verify() happens after the test completes, wrapping with waitForAsync() may fix it